src/app/oauth/confirmed/route.ts

```diff
@@ -0,0 +1,119 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { NextResponse } from "next/server";
+import { FxAOAuthClient, getProfileData, getSha1 } from "../../../utils/fxa.js";
+import {
+  getSubscriberByEmail,
+  updateFxAData,
+} from "../../../db/tables/subscribers.js";
+import { addSubscriber } from "../../../db/tables/emailAddresses.js";
+import { getSession } from "../../functions/server/getSession";
+import { getEmailCtaHref, sendEmail } from "../../../utils/email.js";
+import mozlog from "../../../utils/log.js";
+import { UnauthorizedError } from "../../../utils/error.js";
+import { getTemplate } from "../../../views/emails/email2022.js";
+import { signupReportEmailPartial } from "../../../views/emails/emailSignupReport.js";
+import { getL10n, getLocale } from "../../functions/server/l10n";
+import {
+  getBreachIcons,
+  getBreaches,
+} from "../../functions/server/getBreaches";
+import { getBreachesForEmail } from "../../../utils/hibp.js";
+
+const log = mozlog("controllers.auth");
```

Comment:
Q: should this be `"controllers.auth"` or something more specific? Looks like we're using that string 3&times; in 3 different files:

```diff
 git grep -n " mozlog(" | grep -v "://"

 src/app/functions/server/getBreaches.ts:14:const log = mozlog('hibp')
+src/app/oauth/confirmed/route.ts:25:const log = mozlog("controllers.auth");
+src/app/oauth/logout/route.ts:10:const log = mozlog("controllers.auth");
+src/controllers/auth.js:30:const log = mozlog('controllers.auth')
 src/controllers/hibp.js:27:const log = mozlog('controllers.hibp')
 src/db/tables/breaches.js:9:const log = mozlog('DB.breaches')
 src/db/tables/emailAddresses.js:14:const log = mozlog('DB.email_addresses')
 src/db/tables/subscribers.js:12:const log = mozlog('DB.subscribers')
 src/external/onerep.js:9:const log = mozlog('external.onerep')
 src/middleware/error.js:9:const log = mozlog('middleware')
 src/scripts/migrationCleanup.js:16:const log = mozlog('script.migrationCleanup')
 src/utils/email.js:14:const log = mozlog('email-utils')
 src/utils/hibp.js:24:const log = mozlog('hibp')
```

Also looks like we're using "hibp" in 2 places.

Reply:
Yeah, that's because they're currently all in the same file and therefore have that same label, and I didn't want to introduce discrepancies in our logs.

That said, logging is disabled at the moment since mozlog (and in particular, its main dependency that hasn't been updated in six years) appears to not work well with modern module systems (first guess). So we'll have to either fix that or at least reproduce its logging format (see https://mozilla-hub.atlassian.net/browse/MNTOR-1643).